runtime/lua/vim/filetype.lua

```diff
@@ -2489,7 +2489,7 @@ function M.match(args)
 
   -- Finally, check file contents
   if contents or bufnr then
-    contents = contents or M.getlines(bufnr)
+    contents = contents or { unpack(M.getlines(bufnr, 1, 100)), M.getlines(bufnr, -1) }
```

Comment:
Nitpick: `M.getlines(bufnr, -1)` can be simplified to `M.getlines(bufnr)`

Reply:
wouldn't that get _all_ the lines?

Reply:
Yeah I'm dumb, ignore that :smile: